db/schema.rb

```diff
@@ -439,7 +441,7 @@
     t.bigint "contact_inbox_id"
     t.uuid "uuid", default: -> { "gen_random_uuid()" }, null: false
     t.string "identifier"
-    t.datetime "last_activity_at", default: -> { "CURRENT_TIMESTAMP" }, null: false
+    t.datetime "last_activity_at", default: -> { "now()" }, null: false
```

Comment:
why do we need to change this?

Reply:
Set automatically by db migration command.